jax/dtypes.py

```diff
@@ -28,6 +28,7 @@
 import os
 
 import numpy as onp
+import jax.numpy as np
```

Comment:
Thanks for the ping.

I think the problem is a dependency problem, and it has to do with this line: this `dtypes` module needs to be upstream of `jax.numpy`, and so we can't import `jax.numpy` here. Perhaps just use `onp` instead?

Reply:
worked like a charm, thanks @mattjj !

Reply:
@mattjj I don't see a test for `xla.py`, do you know where I should put unit test for this PR ?

Reply:
@mattjj pinging you again in case you have any suggestions for writing tests in this PR :)